types/electron-settings/v2/electron-settings-tests.ts

```diff
@@ -1,4 +1,5 @@
-import * as settings from 'electron-settings';
+import settings = require('electron-settings');
+import Observer = require('electron-settings/lib/observer');
```

Comment:
I see. If you just need the type (and not the actual class at runtime), you can just export the type from the current typing since the type is already there and defined. 

```ts
declare namespace SettingsInstance {
    export type Observer = ElectronSettings.Observer;
}

declare const SettingsInstance: Settings;
export = SettingsInstance;

```

Then you can reference the type from your code as ```settings.Observer```.

This also saves you from the awkward import of the internal ```lib/observer``` module and has no runtime consequences.